docs/release_notes/8_8.md

```diff
@@ -53,80 +53,83 @@ Some highlights and new features from this release are detailed next.
 
 ### Security
 
-Security-related highlights in the latest Rocky Linux 8.8 release are listed below. For a complete list of security related changes, see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-security).
-
 * FIPS mode settings in the kernel have been adjusted to conform to the Federal Information Processing Standard (FIPS) 140-3. This change introduces stricter settings to many cryptographic algorithms, functions, and cipher suites.
-* `libreswan` packages have been rebased to version 4.9.
+* `libreswan` packages have been rebased to version 4.9
 * You can now filter the RPM database with `fapolicyd` packages.
-* `openscap` packages were rebased to version 1.3.7.
+* `openscap` packages were rebased to version 1.3.7
 * OpenSSL driver can now use certificate chains in Rsyslog.
 * **systemd-socket-proxyd** service now runs in its own SELinux domain due to an update to the SELinux policy.
 
-### Dynamic programming languages, web and database servers
+For a complete list of security-related changes in Rocky Linux 8.8, see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-security).
 
-For a detailed list of the changes in this category see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-dynamic-programming-languages-web-and-database-servers).
+### Dynamic programming languages, web and database servers
 
-More recent versions of the following Application Streams are now available:
+The latest version of the following application module streams is now available:
 
-* nginx 1.22
-* PostgreSQL 15
-* Jakarta XML Binding (`jaxb`) 4
```

Comment:
I still think it's confusing to put one part of the new module streams in one place and one in another. I get the move to the dnf line notation, but also don't see it necessary. (also rather confusing for myself)